device/device.go

```diff
@@ -19,12 +19,18 @@ import (
 	"errors"
 	"fmt"
 	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
 
 	"github.com/astarte-platform/astarte-go/client"
 	"github.com/astarte-platform/astarte-go/interfaces"
 	"github.com/astarte-platform/astarte-go/misc"
 	backoff "github.com/cenkalti/backoff/v4"
 	mqtt "github.com/eclipse/paho.mqtt.golang"
+	_ "github.com/mattn/go-sqlite3"
```

Comment:
Now that you're using gorm (btw, good idea), I think this (and the go module that comes with it) is no longer necessary.